dotcom-rendering/src/web/components/SignInGate/displayRule.ts

```diff
@@ -33,18 +33,16 @@ export const isIOS9 = (): boolean => {
 };
 
 // hide the sign in gate on article types that are not supported
-export const isValidContentType = (CAPI: CAPIBrowserType): boolean => {
+export const isValidContentType = (contentType: string): boolean => {
 	// It's safer to definitively *include* types as we
 	// know new types will not break the sign-in-gate going forward
 	const validTypes = ['Article'];
 
-	return validTypes.some(
-		(type: string): boolean => CAPI.contentType === type,
-	);
+	return validTypes.some((type: string): boolean => contentType === type);
 };
 
 // hide the sign in gate on certain sections of the site, e.g info, about, help etc.
-export const isValidSection = (CAPI: CAPIBrowserType): boolean => {
+export const isValidSection = (sectionName?: string): boolean => {
```

Comment:
I appreciate this is existing logic, but I'm pretty sure we should be using `section` not `sectionName`.
E.g. 
![Screen Shot 2021-11-16 at 09 38 13](https://user-images.githubusercontent.com/1513454/141960383-0439df43-30e6-4256-9f4c-351adc86e0ec.png)

Reply:
I've been migrating to `section` for epics - https://github.com/guardian/dotcom-rendering/pull/3623
This was clearly a mistake

Reply:
Happy to change this! Always keen to leave things in a better state

Reply:
Hmm, so this isn't as simple as you'd think. We don't have `section` as a typed property in DCR, only `sectionName`. We can add it to the type but that needs investigation and possibly some work further up the stack so might be better as a separate PR.

Note. It _does_ exist on the config object but it's untyped. It's sort of there by chance but can't be depended on.

Reply:
Ignore my last, it _is_ typed on `config`. I've also made a note to look at improving our type to include `sectionId`

Reply:
lol, so it is _typed_ but isn't actually there. @tjmw pointed out to me that `section` is actually available on `window.guardian.config.page.section`

<img width="311" alt="Screenshot 2021-11-16 at 12 32 01" src="https://user-images.githubusercontent.com/1336821/141986032-163b43e1-4101-4ad7-b310-20fad643d0df.png">

So. In short, we need to revisit this type and work out how we're going to do things. Which I think should happen in a separate PR.

I suspect this also affect https://github.com/guardian/dotcom-rendering/pull/3623? @tomrf1

Reply:
Well it worked when I tested it! Not sure how it gets into `CAPI.config.section`, but it's there

Reply:
That's a worry. It could be it is being set on some articles and not others? Or it might be it's available on the CAPI server object but isn't passed into the client object?

Reply:
It's apparently coming from here: https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/index.d.ts#L745
How does this get constructed?

Reply:
The data in `window.guardian.config.page` seems to correspond to that model, `ConfigTypeBrowser`, so I'm not sure what the issue is?

Reply:
I had a dig into this and I think I see what's going on 🕵️ 

So the value we need, `section` (or `sectionId`) is - [according to our types](https://github.com/guardian/dotcom-rendering/blob/150fc2d81e6a66d9c3336185e874fc8cd0288546/dotcom-rendering/src/model/window-guardian.ts#L161) - available on `window.guardian.config.section`

But in reality is actually stored on `window.guardian.config.page.section` which is not typed anywhere.

<img width="338" alt="Screenshot 2021-11-16 at 14 51 24" src="https://user-images.githubusercontent.com/1336821/142008031-636ac41b-a081-455c-aa02-238b0c5d7d0a.png">

The reason this data is appearing on `window.guardian.config` without being typed is this code

<img width="767" alt="Screenshot 2021-11-16 at 14 48 46" src="https://user-images.githubusercontent.com/1336821/142008586-56d955cf-a680-499a-bb49-6e217f119dae.png">

We extend the object sent to us from CAPI, without typing it 😐 

Why? Because we have these contracts with the commercial code that expects certain fields on the `window.config` object. We're not sure exactly what these contracts are so we pass everything through.

The commercial team are working on this. Once we have this contract better defined we can dramatically reduce the size of `window.config`. But until then the `section` value is out in the wild west!